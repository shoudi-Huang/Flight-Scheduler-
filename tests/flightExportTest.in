location import locations.csv
flights
flight export
flight export User/error_locationExport.csv
flight export flightExportEmptyTest.csv
flight add sunday 20:00 Berlin NewYork 250
Flight EXPorT flightExport1Test.csv extra parameter kl
flight 0 remove
flight import flights.csv
flight export flightExportTest.csv
exit
