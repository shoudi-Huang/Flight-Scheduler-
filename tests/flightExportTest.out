User: Imported 51 locations.

User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
(None)

User: Error writing file.

User: Error writing file.

User: Exported 0 flights.

User: Successfully added Flight 0.

User: Exported 1 flight.

User: Removed Flight 0, Sun 20:00 Berlin --> NewYork, from the flight schedule.

User: Imported 169 flights.
13 lines were invalid.

User: Exported 169 flights.

User: Application closed.
